### ISSUE TITLE
Ability to Have Zones where loot does not spawn?

### ISSUE THREAD

**Issue by RealFlaw** [dodzh/lootsystem#2]
I Would like to have essentially a safezone where i could make loot not spawn? Hopefully this is possible?

**Comment by dodzh** [comment]
it has been a while since I've used this, but if I remember correctly you define your own zones, so yeah.
love to see you ppl are using this. any other question feel free, ill do my best to answer them.

**Comment by RealFlaw** [comment]
Yeah, Ill take a look into it and try to figure it out, i didn't see anything at a glance that stood out as obvious to define non-loot zones, but ill take another look and get back to you

**Comment by RealFlaw** [comment]
Haven't been able to figure this out for myself, any pointers you might have?

**Comment by dodzh** [comment]
ok so you can do something like this, in the file "LSS_fnc_spawn_loot.sqf" where you see the line "if (!isServer) exitWith {};"
right under it you can add something like:

if ((myMarkers findIf { player inArea _x }) > 0)  exitWith {};

"myMarkers" being an array that contains zone where you dont want the loot to spawn at:

please keep in mind I no longer have Arma 3 installed so im doing this code in my head so play a bit with it. basically you want a single line of code that checks and see if the player is in within defined areas and if is true exits the function
